.github/workflows/builds.yml

```diff
@@ -22,6 +22,9 @@ jobs:
       uses: cake-build/cake-action@v1.4.0
       with:
         target: Test
+
```

Comment:
I'd prefer to create a separate file editorconfig.yml with
```
on:
  pull_request:
    branches: [ main ]
```
because it doesn't make sense to run it 4 times (1 build-windows on PR +1 build-linux on PR + 1 build-windows on merge to main +  1 build-linux on merge to main)